docs/components/buttongroup/index.md

```diff
@@ -1,27 +1,31 @@
 <script setup>
   import Vue from './vue.md';
-  import Elements from './elements.md';
-  import React from './react.md';
 </script>
 
 # Button Group
 
 Button Group is used to group and align clickable elements, applying different visuals for different needs.
 
+<components-status vue='released' />
 ## Example
 
 <theme-switcher />
 
 <buttongroup-example />
 
+## Usage
+This component is only supported in Warp Vue at the moment.
+
+For a React equivalent that serves a similar purpose, use the `Toggle` in [RadioButtons](/components/radiobuttons), and use the `Toggle` with `radio-button` type.
```

Comment:
Just a small thing to prevent a deadlink error. It should be `/components/radiobuttons/`.
<img width="868" alt="Screenshot 2023-06-05 at 08 57 48" src="https://github.com/warp-ds/tech-docs/assets/41303231/1a5b6c7f-e6e4-4d6d-beca-5b84dab62ba0">

Reply:
Also, I think we can skip the part after the link because that documentation will explain what `type` needs to be applied on Toggle.

Reply:
nice, I'll fix